build.psm1

```diff
@@ -1862,6 +1862,18 @@ function New-MSIPackage
         [Switch] $Force
     )
 
+    ## need RCEdit to modify the binaries embedded resources
+    if (-not (Test-Path "~/.rcedit/rcedit-x64.exe"))
```

Comment:
What about Unix?

Reply:
I'm not as familiar on non-Windows with regards to viewing resources (which Explorer does easily), is this something I can do easily with Ubuntu?  If so, I can move this to the build step rather than packaging.

Reply:
Also rcedit only works on Windows

Reply:
`Get-Item <file> | fl *` show ProductVersion on WSL.

Reply:
@iSazonov not working for me with WSL:

```powershell
PS /mnt/c/Windows/System32/WindowsPowerShell/v1.0> get-item ./powershell.exe | fl *


PSPath            : Microsoft.PowerShell.Core\FileSystem::/mnt/c/Windows/System32/WindowsPowerShell/v1.0/powershell.exe
PSParentPath      : Microsoft.PowerShell.Core\FileSystem::/mnt/c/Windows/System32/WindowsPowerShell/v1.0
PSChildName       : powershell.exe
PSDrive           : /
PSProvider        : Microsoft.PowerShell.Core\FileSystem
PSIsContainer     : False
Mode              : --r--l
VersionInfo       : File:             /mnt/c/Windows/System32/WindowsPowerShell/v1.0/powershell.exe
                    InternalName:
                    OriginalFilename:
                    FileVersion:
                    FileDescription:
                    Product:
                    ProductVersion:
```

Reply:
It looks like PowerShell is simply using the FileVersionInfo class which doesn't return anything on Linux:
```
PS /mnt/c/Windows/System32/WindowsPowerShell/v1.0> [System.Diagnostics.FileVersionInfo]::GetVersionInfo("/mnt/c/Windows/System32/WindowsPowerShell/v1.0/powershell.exe")

ProductVersion   FileVersion      FileName
--------------   -----------      --------
                                  /mnt/c/Windows/System32/WindowsPowerShell/v1.0/powershell.exe
```
but on Windows:
```
PS C:\Windows\System32\WindowsPowerShell\v1.0> [System.Diagnostics.FileVersionInfo]::GetVersionInfo("C:\Windows\system32
\WindowsPowerShell\v1.0\powershell.exe")

ProductVersion   FileVersion      FileName
--------------   -----------      --------
10.0.17019.1000  10.0.17019.10... C:\Windows\system32\WindowsPowerShell\v1.0\powershell.exe
```

Reply:
Linux ELF binaries probably don't have a resource section to stuff this info into.

Reply:
@SteveL-MSFT Why you test the Windows FullCRL exe file? In my test I used SMA.dll from PowerShell Core.

Reply:
I used Windows PowerShell.exe as I knew that had that info populated.  I see that SMA.dll from PSCore6 does have that info as does powershell.exe from PSCore6.  So dotnet build is adding that, but the executable doesn't have it.

Reply:
This might be because the executable is an actual ELF binary while the dlls are using the same format as Windows making them portable.